CHANGES.rst

```diff
@@ -258,7 +258,7 @@ astropy.table
   will now always have ``masked=False``, though specific columns will be masked as
   needed. Two new table properties ``has_masked_columns`` and ``has_masked_values``
   were added. See the ``Masking change in astropy 4.0`` section within
-  `<https://docs.astropy.org/en/v4.0/table/masking.html>`_ for details. [#8789]
+  `<https://docs.astropy.org/en/latest/table/masking.html>`_ for details. [#8789]
```

Comment:
This might have ben deliberate btw, since the link might not work anymore once e.g. v5.0 is out

Reply:
Or maybe what we should do is fix it as you suggest and then after the 4.0 release we can change it back.